src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx

```diff
@@ -0,0 +1,1078 @@
+                                    <span>ms</span>
+                                  </td>
+                                </tr>
+                              )}
+                              {this.state.roundTrips !== undefined && (
+                                <tr className="queryMetricsSummaryTuple">
+                                  <td title="Round Trips">Round Trips</td>
+                                  <td>
+                                    <span>{this.state.roundTrips}</span>
+                                  </td>
+                                </tr>
+                              )}
+                              {/* <!-- TODO: Report activity id for mongo queries --> */}
+                              {this.state.activityId !== undefined && (
+                                <tr className="queryMetricsSummaryTuple">
+                                  <td title="Activity id">Activity id</td>
+                                  <td></td>
+                                  <td>
+                                    <span>{this.state.activityId}</span>
+                                  </td>
+                                </tr>
+                              )}
+                            </tbody>
+                          </table>
+                          {this.state.isQueryMetricsEnabled && (
+                            <div className="downloadMetricsLinkContainer">
+                              <a
+                                id="downloadMetricsLink"
+                                role="button"
+                                tabIndex={0}
+                                onClick={() => this.onDownloadQueryMetricsCsvClick()}
+                                onKeyPress={(event: React.KeyboardEvent<HTMLAnchorElement>) =>
+                                  this.onDownloadQueryMetricsCsvKeyPress(event)
+                                }
+                              >
+                                <img
+                                  className="downloadCsvImg"
+                                  src="images/DownloadQuery.svg"
+                                  alt="download query metrics csv"
+                                />
+                                <span>Per-partition query metrics (CSV)</span>
+                              </a>
+                            </div>
+                          )}
+                        </div>
+                      )}
+                      {/* <!-- Query Errors Content - Start--> */}
+                      {!!this.state.error && (
+                        <div className="tab-pane active">
+                          <div className="errorContent">
+                            <span className="errorMessage">{this.state.error}</span>
+                            <span className="errorDetailsLink">
+                              <a
+                                data-bind="click: $parent.onErrorDetailsClick, event: { keypress: $parent.onErrorDetailsKeyPress }"
```

Comment:
Remove or fix. There should be no more data-bind if the component is in React

Reply:
Removed data-bind